gdk_pixbuf2 is now pure G-I

--- origsrc/poppler-3.1.1/ext/poppler/extconf.rb	2017-03-19 18:53:47.253222100 -0500
+++ src/poppler-3.1.1/ext/poppler/extconf.rb	2017-03-19 18:54:21.516318800 -0500
@@ -30,7 +30,7 @@ rescue LoadError
   require 'mkmf-gnome2'
 end
 
-["glib2", "gdk_pixbuf2"].each do |package|
+["glib2"].each do |package|
   directory = "#{package}#{version_suffix}"
   build_dir = "#{directory}/tmp/#{RUBY_PLATFORM}/#{package}/#{RUBY_VERSION}"
   add_depend_package(package, "#{directory}/ext/#{package}",
